src/Handler/Table/Drop/DropTableHandler.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata\Handler\Table\Drop;
+
+use Google\Protobuf\Internal\Message;
+use Keboola\StorageDriver\Command\Table\DropTableCommand;
+use Keboola\StorageDriver\Contract\Driver\Command\DriverCommandHandlerInterface;
+use Keboola\StorageDriver\Credentials\GenericBackendCredentials;
+use Keboola\StorageDriver\Teradata\ConnectionFactory;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+
+final class DropTableHandler implements DriverCommandHandlerInterface
+{
+    /**
+     * @inheritDoc
+     * @param GenericBackendCredentials $credentials
+     */
+    public function __invoke(
+        Message $credentials,
+        Message $command,
+        array $features
+    ): ?Message {
+        assert($credentials instanceof GenericBackendCredentials);
+        assert($command instanceof DropTableCommand);
+
+        $db = ConnectionFactory::getConnection($credentials);
+
+        // TODO path
+        // TODO ignoreErrors
```

Comment:
dobrá poznámka to teď ignoruješ správně path v TD asi zatím nevyužijeme a ignoreErrors sem možná do toho cmd dal zbytečně tady je to enom jeden sql dotaz takže to nedává moc smysl.

Reply:
`ignoreErrors` jsem pouzil nasledovne:
```
// src/Handler/Table/Drop/DropTableHandler.php
        // drop table
        try {
            $db->executeStatement($dropTableSql);
        } catch (Exception $e) {
            if (!$command->getIgnoreErrors()) {
                throw $e;
            }
        }
```